src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -848,6 +886,32 @@ def do_request(params)
         it { expect(assigns(:errors)).not_to be_nil }
         it { expect(response).to have_http_status(:ok) }
       end
+
+      context 'for multibuild package' do
+        let(:params) {
+          { project:    source_project,
+            package:    "#{source_package}:multibuild-package",
+            repository: repo_leap_42_2.name,
+            arch:       architecture.name}
+        }
+
+        context 'instance variables' do
+          before do
+            do_request params
+          end
+
+          # This is fine as backend does not need to run to test this. Otherwise it would be necessary to create first a multibuild package in the backend
```

Comment:
Then please make a better proposal ;) I already asked other people from the team and actually they know what is meant...

Would that be understandable for you?
```
# For this test it is not necessary to have the backend running so the response "No live log available" is fine
```